src/app/@ansyn/map-facade/components/imageries-manager/imageries-manager.component.html

```diff
@@ -25,5 +25,7 @@
 
 <ansyn-context-menu></ansyn-context-menu>
 
+<ansyn-welcome-notification></ansyn-welcome-notification>
```

Comment:
put *ngIf here to create this component if "welcomeNotificatioFlag" was false
on lost focus the flag will be set and the component will be destroyed

Reply:
Done